### PR TITLE
DYN-7647 Search PackageManager EmptySpace

### PR DESCRIPTION
**Pull request by RobertGlobant20**
### Purpose

Fixing Search for packages with empty space in the Package.Name
For searching nodes a fix was implemented for removing empty space in node names when indexing/searching/fetching, then the same fix needs to be applied to Packages name (otherwise if the package being search has empty space in the name, won't be found),

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [X] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [X] The level of testing this PR includes is appropriate
- [X] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB

### Release Notes

Fixing Search for packages with empty space in the Package.Name

### Reviewers

@QilongTang

### FYIs

@reddyashish @zeusongit

### CODE REVIEW

**Comment by RobertGlobant20** [comment]
This is a GIF showing the expected behavior when searching a package with a white space in the name
98JNkXh16S

**Comment by dnenov** [comment]
Oh, I think that was the test that was failing for me lately - I thought I had introduced a breaking change, but I don’t remember touching the search functionality itself… . Apologies if it was me somehow, @RobertGlobant20 !

**Comment by RobertGlobant20** [comment]
> Oh, I think that was the test that was failing for me lately - I thought I had introduced a breaking change, but I don’t remember touching the search functionality itself… . Apologies if it was me somehow, @RobertGlobant20 !

@dnenov do you remember which was the test failing?

**Review by QilongTang — APPROVED** [review]
Anyway we can create a unit test for this?

**Comment by RobertGlobant20** [comment]
> Anyway we can create a unit test for this?

working on it

**Comment by RobertGlobant20** [comment]
> Anyway we can create a unit test for this?

@QilongTang I've added a unit test for this case
commit: https://github.com/DynamoDS/Dynamo/pull/15566/commits/30ab157dc0babb565ea947e355e8d3d1c5a09365

**Review by reddyashish — APPROVED** [review]

File: test/DynamoCoreWpfTests/PackageManager/PackageManagerSearchElementViewModelTests.cs [hunk 1]
```diff
@@ -730,10 +730,11 @@ public void PackageSearchOrderAfterTextReset()
         public void PackageSearchWithWhitespaceInName()
         {
             var packageToBeSearched = "Dynamo Samples";
-            var packagesListNames =  new List<string> { packageToBeSearched, "archi-lab.net", "LunchBox for Dynamo", "DynamoSap", "TuneUp" };
+            var packagesListNames =  new List<string> { "Dynamo Samples", "archi-lab.net", "LunchBox for Dynamo", "DynamoSap", "TuneUp" };
```
Comment (zeusongit) [inline test/DynamoCoreWpfTests/PackageManager/PackageManagerSearchElementViewModelTests.cs:733]: remove variable `packageToBeSearched`?

Comment (RobertGlobant20) [reply test/DynamoCoreWpfTests/PackageManager/PackageManagerSearchElementViewModelTests.cs:733]: The packageToBeSearched was removed

**Comment by RobertGlobant20** [comment]
Same job passing in master-15 without failures
https://master-15.jenkins.autodesk.com/view/DYN/job/DYN-DevCI_Self_Service/1449/

**Pull request merged**